source/_integrations/opnsense.markdown

```diff
@@ -21,53 +22,17 @@ within Home Assistant:
 
 - [Presence detection](#presence-detection)
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To configure OPNsense integration with Home Assistant add the following section
-to your configuration.yaml:
-
-```yaml
-opnsense:
-  url: https://router/api
-  api_secret: API_SECRET
-  api_key: API_KEY
-```
-
-Where the `api_key` and `api_secret` values are acquired from your OPNsense
+The API key and API secret values are acquired from your OPNsense
 router using the web interface. For more information on this procedure, refer
 to the OPNsense [documentation](https://docs.opnsense.org/development/how-tos/api.html#creating-keys).
 
-User with API Key requires privileges for Type: 
+User with API Key requires privileges for Type:
 
 - GUI Name: Diagnostics: ARP Table
 - GUI Name: Diagnostics: Network Insight
 
-{% configuration %}
```

Comment:
Is the configuration information mentioned here entered via UI? In that case, you could keep the information but convert it to the [configuration_basic format](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L57).